File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] a) {
        int n = a.length;
        for(int i = 0 ; i < n ; i ++){
            boolean ok = true ;
            for(int j = 1 ; j < n ; j ++){
                if(a[(i + j) % n] < a[(i + j - 1 + n) % n]){
                    ok = false ;
                    break ;
                }
            }
            if(ok){
                return true ;
            }
        }
        return false ;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n = len(nums)
        for i in range(n):
            ok = True
            for j in range(1, n):
                if nums[(j + i) % n] < nums[(j + i - 1 + n) % n]:
                    ok = False 
                    break 
            if ok:
                return True
            
        return False 

